bin/MadGraph5_aMCatNLO/cards/production/13p6TeV/thq_4f_ckm_LO_ctcvcp/thq_4f_ckm_LO_ctcvcp_proc_card.dat

```diff
@@ -0,0 +1,10 @@
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model sm
```

Comment:
these could be removed

Reply:
removed but do we need to als removed last line "import model sm", just to confirm.